docs/source/guides/logging.md

```diff
@@ -0,0 +1,65 @@
+# Logging
+
+Anomalib offers various mechanisms for logging metrics and predicted masks.
+
+## Enabling Logging
+
+These can be enabled using the `logger` parameter in `project` section of each model configuration file. The available options are `tensorboard`, `wandb` and `csv`.
+For example, to log to TensorBoard:
```

Comment:
this PR changes this right? `logger` parameter will be in `logging` section of the config file?

Reply:
Yeah. I'll update the text if you guys are satisfied with the new design.